lib/manageiq/providers/amazon/engine.rb

```diff
@@ -13,6 +13,14 @@ def self.vmdb_plugin?
         def self.plugin_name
           _('Amazon Provider')
         end
+
+        def self.init_loggers
+          $aws_log ||= Vmdb::Loggers.create_logger("aws.log")
```

Comment:
The reason I have `||=` here is that something else could initialize the logger first.  For example, in core, FileDepotS3 needs $aws_log, so we might need to initialize in core anyway.  Similarly, multiple providers might use $fog_log, and each would try to initialize.

Reply:
Do we have to be concerned about this being initialized in multiple threads/processes at all?

Reply:
I don't believe so.